src/main/webapp/app/shared/util/fullscreen.util.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * checks if this component is the current fullscreen component
+ */
+export function isFullScreen() {
+    const docElement = document as any;
+    // check if this component is the current fullscreen component for different browser types
+    if (docElement.fullscreenElement !== undefined) {
+        return docElement.fullscreenElement;
+    } else if (docElement.webkitFullscreenElement !== undefined) {
+        return docElement.webkitFullscreenElement;
+    } else if (docElement.mozFullScreenElement !== undefined) {
+        return docElement.mozFullScreenElement;
+    } else if (docElement.msFullscreenElement !== undefined) {
+        return docElement.msFullscreenElement;
+    }
```

Comment:
an alternative return statement of false would be good here, I suppose. 

```suggestion
    }
    return false;
```

Reply:
Done